commit 1d008ef39daacdaf97d87a482a334d31d3ef094e (refs/changes/59/659/2)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2016-10-12T18:52:10+00:00 (3 years ago)
    
    ADMINUI-2345 Validation of imgapi external nic availability does not filter out deleted instances

diff --git a/lib/api/imgapi-external-nic-status.js b/lib/api/imgapi-external-nic-status.js
index 962bf787..7466b578 100644
--- a/lib/api/imgapi-external-nic-status.js
+++ b/lib/api/imgapi-external-nic-status.js
@@ -12,6 +12,7 @@ var restify = require('restify');
 
 module.exports = function imgapiExternalNicStatus(req, res, next) {
     req.sdc[req.dc].vmapi.listVms({
+        'state':'active',
         'tag.smartdc_role':'imgapi',
         'tag.smartdc_type':'core'
     }, function listVmsCallback(err, vms) {
